llm4pddl/approaches/base_approach.py

```diff
@@ -14,6 +14,7 @@ class BaseApproach(abc.ABC):
 
     def __init__(self) -> None:
         self._rng = np.random.default_rng(FLAGS.seed)
+        self.embeddings_mapping: List[Dict] = []
```

Comment:
I don't think _all_ approaches need embeddings (e.g., pure planning approaches certainly don't) -- can we just define this within the LLMOpenLoopApproach?